metrics/metrics-core/src/test/java/org/apache/servicecomb/metrics/core/TestHealthCheckerPublisher.java

```diff
@@ -30,8 +30,7 @@
 
   @Test
   public void test() {
-    HealthCheckerManager manager = new HealthCheckerManager();
-    manager.register(new HealthChecker() {
+    HealthCheckerManager.getInstance().register(new HealthChecker() {
```

Comment:
Here you just check one HealthChecker. The test should test at least zero，one and two HealthChecker.

Reply:
Done

Reply:
zero，one and two HealthChecker check logic had added in TestHealthCheckerManager